lib/widgets/payment_details_dialog.dart

```diff
@@ -278,6 +284,18 @@ List<Widget> _getStreamedPaymentInfoDetails(StreamedPaymentInfo paymentInfo) {
   }).toList();
 }
 
+List<Widget> _getLNUrlSuccessActionForPayment(
+    BuildContext context, SuccessAction sa) {
+  return <Widget>[
+    if (sa.tag == 'url')
+      ShareablePaymentRow(title: "Description", sharedValue: sa.description),
+    if (sa.tag == 'url')
+      ShareablePaymentRow(title: "URL", sharedValue: sa.url), // TODO Hyperlink.
```

Comment:
Why do you need to check twice if `sa.tag == 'url'`?

Reply:
I didn't know how to use spread syntax in conditional list construction. Now I know. Will fix.